style.css

```diff
@@ -0,0 +1,51 @@
+body {
+  background-image: url(./images/header-illsutration-mobile@2x.png);
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  margin: auto;
+}
+
+.flex-container {
+  margin: 20px;
+  display: flex;
+  justify-content: space-between;
+}
```

Comment:
- [ ] In order to match the given design please add a white background to navbar and align items also vertically. 

Here's what you need to build:
![image](https://user-images.githubusercontent.com/91486791/198370840-c6816a71-779e-465c-9c5d-7dd45b17796c.png)

Here's what you build: 
![image](https://user-images.githubusercontent.com/91486791/198370778-adceff5f-65d3-4136-a474-5291fb668d13.png)

Reply:
Fixed